src/XliffTasks/Tasks/GatherXlf.cs

```diff
@@ -32,10 +32,11 @@ protected override void ExecuteCore()
             foreach (var source in Sources)
             {
                 string sourcePath = source.ItemSpec;
+                string sourceDocumentPath = source.GetMetadataOrDefault(MetadataKey.SourceDocumentPath, source.ItemSpec);
```

Comment:
We should remove sourcePath variable and use sourceDocumentPath in error below. I think that's the only usage.

Reply:
Done.  Updates pushed.